e2e_tests/tests/cluster/test_checkpoints.py

```diff
@@ -47,7 +47,6 @@ def wait_for_gc_to_finish(sess: api.Session, experiment_ids: List[int]) -> None:
 
 
 @pytest.mark.e2e_gpu
-@pytest.mark.e2e_slurm_gpu
 def test_set_gc_policy() -> None:
```

Comment:
wut.  Why was this ever a gpu test.  Seriously.

Instead of demoting this test to e2e_slurm, I moved the e2e_slurm to test_delete_checkpoints which actually tests that checkpoint_gc works.

(I almost deleted `test_set_gc_policy()` last week just because it doesn't check the results of setting gc policy, it only makes sure the CLI doesn't crash(!))